services/core/methods/controllers/accounts.js

```diff
@@ -43,7 +43,12 @@ const getKnownAccounts = async () => {
 };
 
 const getDataForAccounts = async params => {
-	const accounts = await CoreService.getAccounts(params);
+	let accounts;
+	if (params.isDelegate === 'true') {
+		accounts = await CoreService.getDelegates(params);
+	} else {
+		accounts = await CoreService.getAccounts(params);
+	}
```

Comment:
Gateway should map `params.isDelegate` as a `boolean` as per the current configuration in `services/gateway/apis/http-version2/methods/accounts.js`. This condition must always be resolving to `false` as the comparison is against a string (`'true'`). Please confirm again.

```suggestion
	accounts = params.isDelegate
		? await CoreService.getDelegates(params);
		: await CoreService.getAccounts(params);
```